### ISSUE TITLE
Does it support https ?

### ISSUE THREAD

**Issue by pranipat** [h4cc/wkhtmltopdf-amd64#32]
It works fine with http urls but not with https url ? What is the solution for that ?

**Comment by barryvdh** [comment]
Updating to 0.12.5 should fix that, once it's tagged.

**Comment by stevro** [comment]
0.12.5 is tagged :)
https://github.com/wkhtmltopdf/wkhtmltopdf/releases/tag/0.12.5

**Comment by barryvdh** [comment]
Yeah I meant, when this repository publishes released with that version tagged :)
If you install it from the website instead of with Composer, it works fine.

**Comment by tristanbes** [comment]
Can we get a release with 0.12.5 binaries please with this packages ? ping @h4cc 
Otherwise we can't use it in HTTPS context :s

**Comment by clemblanco** [comment]
ping @h4cc

**Comment by marijnotte** [comment]
@h4cc could you please release a new version? Would be much appreciated!

**Comment by grzegorztomasiak** [comment]
+1 would be nice to have 0.12.5 version released

**Comment by politsin** [comment]
+1

**Comment by fromz** [comment]
I wonder if h4cc is still around

**Comment by benjaminniess** [comment]
Hello there,

Any news of this 0.12.5 tag?
Thanks

**Comment by gnoesiboe** [comment]
?

**Comment by marcus-at-localhost** [comment]
I found a 12.5 static build here: http://www.ajvg.com/wkthmltopdf_static_build_for_Debian_based_Linux_download.htm - but it didn't solve the https problem :-( - Why is converting to pdf so hard?
